src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -1701,4 +1701,10 @@ public ColonyState getState()
     {
         return colonyStateMachine.getState();
     }
+
+    @Override
+    public boolean isActive()
+    {
+        return colonyStateMachine.getState() != INACTIVE;
```

Comment:
why not == ACTIVE?

Reply:
Because there were some other states too, that would be fine.

Reply:
ye there is UNLOADED which is the state when the colony doesnt meet the required amount of loaded chunks

Reply:
Yeah, I wanted citizens that are loaded and the player is online still to consume food, etc

Reply:
kk then it does what you want